apps/user-service/src/main/resources/application.yaml

```
@@ -15,3 +15,11 @@ eureka:
   client:
     service-url:
       defaultZone: http://localhost:8761/eureka/
+
+application:
+  security:
+    jwt:
+      secret: ${secrets.security.jwt.secret}
+      expiration: ${secrets.security.jwt.expiration}
+  bcrypt:
+    strength: ${secrets.bcrypt.strength}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)